lib/app/admin/router.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var rebuildData = require('./data-reset');
+var HttpStatus = require('http-status-codes');
+
+
+function getResetDataHandler(mediator) {
+
+  return function resetData(req, res) {
+    rebuildData(mediator, req.body.remove_list)
+      .then(function() {
+        res.send(HttpStatus.NO_CONTENT);
```

Comment:
`res.status(HttpStatus.NO_CONTENT).end()`

`res.send(HttpStatus.NO_CONTENT);` will actually send 204 in the response with a HTTP Status of 200.